vllm/config/compilation.py

```diff
@@ -192,7 +192,7 @@ class CompilationConfig:
     used for the compilation directly (it sees the whole graph). When the
     compilation level is 3, the backend is used for the piecewise compilation
```

Comment:
Note that the backend cannot be custom for level 3, and also add that compilation is only piecewise if splitting ops are set (by default) and `use_inductor_cudagraph_partition` is off